### PR TITLE
Copy OSS tracing content to Enterprise 1.9

### LINKED ISSUE #3455: Copy tracing content from InfluxDB OSS to Enterprise
There is currently a pointer from the Enterprise docs to OSS for details re: tracing.  That link is currently broken.

The resolution to this, however, may just be to copy the content from OSS to Enterprise so that we can continue to enhance this for Enterprise customers going forward.

##### Relevant URLs
Enterprise docs where the tracing tag (very bottom of the page) is broken/not directing to the right location: https://docs.influxdata.com/enterprise_influxdb/v1.9/administration/monitor/logs/
OSS content: https://docs.influxdata.com/influxdb/v1.8/administration/logs/#tracing

### PR DESCRIPTION
**Pull request by pierwill**
Closes #3455.

### CODE REVIEW

File: content/enterprise_influxdb/v1.9/administration/monitor/logs.md [hunk 1]
```diff
@@ -130,6 +130,181 @@ See [Structured logging](/enterprise_influxdb/v1.9/administration/logs/#structur
...
+For example, you can grep by values in `op_name` AND `op_event` to find all starting operation log entries.
+For an example of this, see [Finding all starting log entries](#finding-all-starting-operation-log-entries).
+
+**Example:** `op_event=start`
+
+##### `op_elapsed`
+Amount of time the operation spent executing.
+Logged with the ending trace log entry.
+Time unit displayed depends on how much time has elapsed -- if it was seconds, it will be suffixed with `s`.
+Valid time units are `ns`, `µs`, `ms`, and `s`.
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.9/administration/monitor/logs.md:170]:
```suggestion
Duration of the operation execution.
Logged with the ending trace log entry.
Valid duration units are `ns`, `µs`, `ms`, and `s`.
```

File: content/enterprise_influxdb/v1.9/administration/monitor/logs.md [hunk 2]
```diff
@@ -130,6 +130,181 @@ See [Structured logging](/enterprise_influxdb/v1.9/administration/logs/#structur
...
+
+**Example:** `op_elapsed=0.352ms`
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.9/administration/monitor/logs.md:172]: This doesn't look like a valid duration (as Flux/InfluxDB generally parses them, but that may be something unique to the tracing data format. I'm not sure. I would expect this to be `op_elapsed=352ms` or `op_elapsed=352µs` (the equivalent of `0.325ms`). Duration values typically don't support fractional numbers.

File: content/enterprise_influxdb/v1.9/administration/monitor/logs.md [hunk 3]
```diff
@@ -130,6 +130,181 @@ See [Structured logging](/enterprise_influxdb/v1.9/administration/logs/#structur
...
+
+`db_shard_group` Shard group identifier
+
+### Tooling
+
+Here are a couple of popular tools available for processing and filtering log files output in `logfmt` or `json` formats.
+
+#### hutils
+
+The [hutils](https://blog.heroku.com/hutils-explore-your-structured-data-logs), provided by Heroku, is a collection of command line utilities for working with logs with `logfmt` encoding, including:
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.9/administration/monitor/logs.md:198]:
```suggestion
The [hutils](https://blog.heroku.com/hutils-explore-your-structured-data-logs) utility collection, provided by Heroku, provides tools for working with `logfmt`-encoded logs, including:
```

File: content/enterprise_influxdb/v1.9/administration/monitor/logs.md [hunk 4]
```diff
@@ -130,6 +130,181 @@ See [Structured logging](/enterprise_influxdb/v1.9/administration/logs/#structur
...
+
+* `lcut`: Extracts values from a `logfmt` trace based on a specified field name.
+* `lfmt`: Prettifies `logfmt` lines as they emerge from a stream, and highlights their key sections.
+* `ltap`: Accesses messages from log providers in a consistent way to allow easy parsing by other utilities that operate on `logfmt` traces.
+* `lviz`: Visualizes `logfmt` output by building a tree out of a dataset combining common sets of key-value pairs into shared parent nodes.
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.9/administration/monitor/logs.md:203]:
```suggestion
- **lcut**: Extracts values from a `logfmt` trace based on a specified field name.
- **lfmt**: Prettifies `logfmt` lines as they emerge from a stream, and highlights their key sections.
- **ltap**: Accesses messages from log providers in a consistent way to allow easy parsing by other utilities that operate on `logfmt` traces.
- **lviz**: Visualizes `logfmt` output by building a tree out of a dataset combining common sets of key-value pairs into shared parent nodes.
```

File: content/enterprise_influxdb/v1.9/administration/monitor/logs.md [hunk 5]
```diff
@@ -130,6 +130,181 @@ See [Structured logging](/enterprise_influxdb/v1.9/administration/logs/#structur
...
+The lnav viewer provides a single log view, automatic log format detection, filtering, timeline view, pretty-print view, and querying logs using SQL.
+
+### Operations
+
+The following operations, listed by their operation name (`op_name`) are traced in InfluxDB internal logs and available for use without changes in logging level.
+
+#### Initial opening of data files
+
+The `tsdb_open` operation traces include all events related to the initial opening of the `tsdb_store`.
+
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.9/administration/monitor/logs.md:217]:
```suggestion
`tsdb_open` operation traces include all events related to the initial opening of the `tsdb_store`.
```

File: content/enterprise_influxdb/v1.9/administration/monitor/logs.md [hunk 6]
```diff
@@ -130,6 +130,181 @@ See [Structured logging](/enterprise_influxdb/v1.9/administration/logs/#structur
...
+The `tsm1_cache_snapshot` operation represents the snapshotting of the TSM in-memory cache to disk.
+
+#### TSM compaction strategies
+
+The `tsm1_compact_group` operation includes all trace log entries related to TSM compaction strategies and displays the related TSM compaction strategy keys:
+
+* `tsm1_strategy`: `level` | `full`
+* `tsm1_level`: `1` | `2` | `3`
+* `tsm1_optimize`: `true` | `false`
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.9/administration/monitor/logs.md:233]:
```suggestion
- **tsm1\_strategy**: level or full
- **tsm1\_level**: 1, 2, or 3
- **tsm\_optimize**: true or false
```

File: content/enterprise_influxdb/v1.9/administration/monitor/logs.md [hunk 7]
```diff
@@ -130,6 +130,181 @@ See [Structured logging](/enterprise_influxdb/v1.9/administration/logs/#structur
...
+
+The `series_partition_compaction` operation includes all trace log entries related to series file compactions.
+
+####  Continuous query execution (if logging enabled)
+
+The `continuous_querier_execute` operation includes all continuous query executions, if logging is enabled.
+
+#### TSI log file compaction
+
+The `tsi1_compact_log_file`
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.9/administration/monitor/logs.md:245]: Missing a description here.

File: content/enterprise_influxdb/v1.9/administration/monitor/logs.md [hunk 8]
```diff
@@ -130,6 +130,181 @@ See [Structured logging](/enterprise_influxdb/v1.9/administration/logs/#structur
...
+The `tsi1_compact_to_level` operation includes all trace log entries for TSI level compactions.
+
+
+### Tracing examples
+
+#### Finding all trace log entries for an InfluxDB operation
+
+In the example below, you can see the log entries for all trace operations related to a "TSM compaction" process.
+Note that the initial entry shows the message "TSM compaction (start)" and the final entry displays the message "TSM compaction (end)".
+\[Note: Log entries were grepped using the `trace_id` value and then the specified key values were displayed using `lcut` (an hutils tool).\]
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.9/administration/monitor/logs.md:258]:
```suggestion
{{% note %}}
 Log entries were grepped using the `trace_id` value and then the specified key values were displayed using `lcut` (an `hutils` tool).
 {{% /note %}}\]
```

**Review by sanderson — CHANGES_REQUESTED** [review]

File: content/enterprise_influxdb/v1.9/administration/monitor/logs.md (hunk 2 above)
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/administration/monitor/logs.md:172]:
```suggestion
**Example:** `op_elapsed=352ms`
```

File: content/enterprise_influxdb/v1.9/administration/monitor/logs.md [hunk 9]
```diff
@@ -130,6 +130,179 @@ See [Structured logging](/enterprise_influxdb/v1.9/administration/logs/#structur
...
+
+The `series_partition_compaction` operation includes all trace log entries related to series file compactions.
+
+####  Continuous query execution (if logging enabled)
+
+The `continuous_querier_execute` operation includes all continuous query executions, if logging is enabled.
+
+#### TSI log file compaction
+
+The `tsi1_compact_log_file`
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/administration/monitor/logs.md:241]:
```suggestion
The `tsi1_compact_log_file` operation includes all trace log entries related to log file compactions.
```

**Review by sanderson — APPROVED** [review]

**Pull request merged**